File: ThaiTuanERP2025.Client/src/app/pages/account/dtos/group.dto.ts
import { UserDto } from "./user.dto";

export interface GroupDto {
      id: string;
      name: string;
      description: string;
      adminId: string;
      adminName: string;
      memberCount: number;
      members: UserDto[];
}

export interface CreateGroupDto {
      name: string;
      description: string;
      adminUserId: string;
}

File: ThaiTuanERP2025.Server/ThaiTuanERP2025.Application/Account/Dtos/GroupDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ThaiTuanERP2025.Application.Account.Dtos
{
	public class GroupDto
	{
		public Guid Id { get; set; }
		public string Name { get; set; } = string.Empty;
		public string Description { get; set; } = string.Empty;
		public Guid AdminId { get; set; }	
		public string AdminName { get; set; } = string.Empty;
		public int MemberCount { get; set; } = 0;
		public List<UserDto> Members { get; set; } = new();
	}
}
